yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -136,10 +137,22 @@ export class Archiver implements L2BlockSource, UnverifiedDataSource, ContractDa
       currentBlockNumber,
       this.lastProcessedBlockNumber + 1n, // + 1 to prevent re including messages from the last processed block
     );
+    const retrievedCancelledL1ToL2Messages = await retrieveNewCancelledL1ToL2Messages(
+      this.publicClient,
+      this.inboxAddress,
+      blockUntilSynced,
+      currentBlockNumber,
+      this.nextL2BlockFromBlock + 1n,
+    );
+
     // TODO: optimise this - there could be messages in confirmed that are also in pending. No need to modify storage then.
```

Comment:
Is this todo tracked somewhere or stale? 👀

Reply:
Just created!